src/platform/test.rs

```diff
@@ -770,3 +770,101 @@ mod sync_test {
         platform::OsIpcSender::test_not_sync();
     }
 }
+
+// Note! This test is actually used by the
+// cross_process_sender_transfer_spawn() test below.  Running it by
+// itself is meaningless.
+#[cfg(not(any(feature = "force-inprocess", target_os = "android")))]
+#[test]
+#[ignore]
+fn cross_process_two_step_transfer_server()
+{
+    let cookie: &[u8] = b"cookie";
+    let mut channel_name: Option<String> = None;
+
+    for arg in env::args() {
+        if arg.starts_with("channel_name:") {
+            let (_, name) = arg.split_at("channel_name:".len());
+            channel_name = Some(String::from(name));
```

Comment:
I pulled this out into a helper function and used a subslice. Not worth doing anything more clever for testing code.